5.5/root/usr/bin/run-mysqld

```diff
@@ -14,10 +16,18 @@ else
   start_local_mysql "$@"
 fi
 
-[ -f ${CONTAINER_SCRIPTS_PATH}/passwd-change.sh ] && . ${CONTAINER_SCRIPTS_PATH}/passwd-change.sh
-[ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ] && . ${CONTAINER_SCRIPTS_PATH}/post-init.sh
+if [ -f ${CONTAINER_SCRIPTS_PATH}/passwd-change.sh ]; then
+  log_info 'Setting passwords ...'
+  . ${CONTAINER_SCRIPTS_PATH}/passwd-change.sh
+fi
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ]; then
+  log_info 'Sourcing post-init.sh ...'
```

Comment:
What does this message mean for the user? Wouldn't be better to explain what post-init do here?